### DISCUSSION TITLE
Any reason why Gopls (go language server) is not installed by default with the extra configuration?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#2716] [Q&A] [answered]
Here is the place where I think it should be https://github.com/LazyVim/LazyVim/blob/cca2e091490fc87eafd7ee27a965e4cbcacdc012/lua/lazyvim/plugins/extras/lang/go.lua#L89

I installed the extra but then I had to go to mason and install it manually.

If is just a mistake I will create a PR, just checking with @folke

**Comment by dpetka2001** [comment]
I don't do Go, but I just installed the Extra and the `gopls` server got installed automatically.

**Reply by Raagh** [reply]
strange, that didnt happened for me, I had to manually install it....

**Reply by dpetka2001** [reply, marked as answer]
I just tried it with a Lazyvim fresh starter template. Try the following
```bash
1. `mkdir ~/.config/lazytest`
2. `git clone https://github.com/LazyVim/starter ~/.config/lazytest`
3. `NVIM_APPNAME=lazytest nvim` and wait for the installation to complete
4. Restart Neovim with the command provided in step 3
5. Enable the LazyExtra for go and restart Neovim with command from step 3
6. Open a file/buffer
```
You should see `gopls` being installed in the background via a notification being popped up. You can also verify it by opening Mason UI by pressing `<leader>cm`.
For cleanup
```bash
rm -rf ~/.config/lazytest
rm -rf ~/.local/share/lazytest
rm -rf ~/.local/state/lazytest
rm -rf ~/.cache/lazytest
```

**Reply by Raagh** [reply]
I think this might have happened because I tried to install the extras before having go installed (new machine), probably the language server is being installed with go install

**Comment by waflessnet** [comment]
Hi, 

i am try lazytest but not work. 

i try install using LaxyExtras 

![image](https://github.com/LazyVim/LazyVim/assets/6413704/e18c9223-6b35-4538-9572-482e1e210bf1)


check log mason. 


![image](https://github.com/LazyVim/LazyVim/assets/6413704/4628b99d-ed2a-4976-ba68-22795cf24e43)




> I just tried it with a Lazyvim fresh starter template. Try the following
> 
> ```shell
> 1. `mkdir ~/.config/lazytest`
> 2. `git clone https://github.com/LazyVim/starter ~/.config/lazytest`
> 3. `NVIM_APPNAME=lazytest nvim` and wait for the installation to complete
> 4. Restart Neovim with the command provided in step 3
> 5. Enable the LazyExtra for go and restart Neovim with command from step 3
> 6. Open a file/buffer
> ```
> 
> You should see `gopls` being installed in the background via a notification being popped up. You can also verify it by opening Mason UI by pressing `<leader>cm`. For cleanup
> 
> ```shell
> rm -rf ~/.config/lazytest
> rm -rf ~/.local/share/lazytest
> rm -rf ~/.local/state/lazytest
> rm -rf ~/.cache/lazytest
> ```

**Reply by waflessnet** [reply]
Ok 

my error was go1.19 then change go1.22.2, work. 

Thanks!.
